test/createTest.m

```diff
@@ -0,0 +1,81 @@
+%==========================================================================
+%This function writes a function, testDataSets, for testing the viewer on
+%all jsons stored locally in the Data/jsons directory. 
+%
+%Authors: Thomas Maullin, Camille Maumet, Thomas Nichols
+%==========================================================================
+
+function createTest()
+       
+    %Create new test file for editing.
+    FID = fopen(fullfile(fileparts(mfilename('fullpath')), 'testDataSets.m'),'wt');
+
+    %Add the start of the test code.
+    start = sprintf('%s' , '%%==========================================================================',...
```

Comment:
@TomMaullin: I don't think this has been solved... Can you add the `test/testDataSets.m` file to git?